### PR TITLE
Adds UpsertServerGroupTags as Atomic Operation and annotates the resp…

### PR DESCRIPTION
**Pull request by ttomsu**
…ective classes.

@duftler PTAL

@spinnaker/reviewers FYI

### CODE REVIEW

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/converters/UpsertGoogleServerGroupTagsAtomicOperationConverter.groovy [hunk 1]
```diff
@@ -16,12 +16,15 @@
...
 
+import com.netflix.spinnaker.clouddriver.google.GoogleOperation
 import com.netflix.spinnaker.clouddriver.google.deploy.description.UpsertGoogleServerGroupTagsDescription
 import com.netflix.spinnaker.clouddriver.google.deploy.ops.UpsertGoogleServerGroupTagsAtomicOperation
 import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperation
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperations
 import com.netflix.spinnaker.clouddriver.security.AbstractAtomicOperationsCredentialsSupport
 import org.springframework.stereotype.Component
 
+@GoogleOperation(AtomicOperations.UPSERT_SERVER_GROUP_TAGS)
```
Comment (duftler) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/converters/UpsertGoogleServerGroupTagsAtomicOperationConverter.groovy]: Can probably drop the `("upsertGoogleServerGroupTagsDescription")` from the `@Component` annotation on the next line.

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/UpsertGoogleServerGroupTagsDescriptionValidator.groovy [hunk 2]
```diff
@@ -17,12 +17,15 @@
...
 import com.netflix.spinnaker.clouddriver.deploy.DescriptionValidator
+import com.netflix.spinnaker.clouddriver.google.GoogleOperation
 import com.netflix.spinnaker.clouddriver.google.deploy.description.UpsertGoogleServerGroupTagsDescription
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperations
 import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Component
 import org.springframework.validation.Errors
 
+@GoogleOperation(AtomicOperations.UPSERT_SERVER_GROUP_TAGS)
```
Comment (duftler) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/UpsertGoogleServerGroupTagsDescriptionValidator.groovy]: Can probably drop the ("upsertGoogleServerGroupTagsDescriptionValidator") from the @Component annotation on the next line.

**Comment by duftler** [comment]
LGTM. Needs a rebase.

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/UpsertGoogleServerGroupTagsDescriptionValidator.groovy (hunk 2 above)
Comment (ttomsu) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/UpsertGoogleServerGroupTagsDescriptionValidator.groovy]: I'll come back and do a sweeping delete of these in another PR.

**Pull request merged**